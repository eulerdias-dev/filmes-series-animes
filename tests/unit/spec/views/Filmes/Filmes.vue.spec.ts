import { FilmesFactory } from './FilmesFactory'

describe('PÃ¡gina de listagem Filmes', () => {
  const factory = new FilmesFactory()

  it('Ao que Titulo seja \'Filmes\' ', async (done) => {
    await factory.montarComponente()
    const tituloEsperado = 'Filmes'

    const tituloObtido = await factory.obterTextoPorId('#titulo')

    expect(tituloEsperado).toEqual(tituloObtido)
    done()
  })

  it('Snapshot', async (done) => {
    await factory.montarComponente()

    expect(factory.componente.html())
      .toMatchSnapshot()
    done()
  })
})
