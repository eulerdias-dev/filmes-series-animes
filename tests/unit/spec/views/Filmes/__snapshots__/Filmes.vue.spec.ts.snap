// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PÃ¡gina de listagem Filmes Snapshot 1`] = `
<v-container>
  <v-row no-gutters="">
    <v-subheader id="titulo">Filmes</v-subheader>
  </v-row>
</v-container>
`;
